README.md

```diff
@@ -1,9 +1,54 @@
-# pm-idm
+# IDM - Identity Manager
 
-Project Management for the IDM Project
+The Identity Manager - IDM for short - is a unified digital wallet based on open-standards that aims to support multiple decentralized identities.
 
-[![#ipfs-identity](https://img.shields.io/badge/irc-%23ipfs--identity-brightgreen.svg)](https://webchat.freenode.net/?channels=ipfs-identity)
+## Documents
 
-## Current status
+Here is a list of some useful documents. If you're interested in collaborating, please check [Contributing](#contributing).
 
-[![Waffle.io - Columns and their card count](https://badge.waffle.io/ipfs-shipyard/pm-idm.svg?columns=all)](https://waffle.io/ipfs-shipyard/pm-idm)
+[💡 Concept](docs/idm-concept.md)
+[🛠 IDM Breakdown](https://docs.google.com/document/d/1g0TjSPjEM4pryPwJTGhIeE4DBsj-VJpz_JqbfllJUgA)
+[🗓 IDM Workplan](https://docs.google.com/spreadsheets/d/1Venqgkcao2Lcje0mkCxr9u0H037aC3H5IxMfsoaeMoE)
```

Comment:
The documents all being displayed in a single line. Should be separate lines.

Reply:
Also add the Information Architecture document (is an image but it's also a document). See: https://github.com/ipfs-shipyard/pm-idm/issues/7#issuecomment-462456778